app/routes/_layout+/settings.template.index.tsx

```diff
@@ -0,0 +1,286 @@
+            {!hasItems ? (
+              <EmptyState
+                customContent={{
+                  title: "No templates on database",
+                  text: "What are you waiting for? Add your first template now!",
+                  newButtonRoute: `new`,
+                  newButtonContent: "Add Template",
+                }}
+                modelName={{
+                  singular: "template",
+                  plural: "templates",
+                }}
+              />
+            ) : (
+              <>
+                <div className="flex w-full items-center justify-between">
+                  <h3 className="text-md text-gray-900">PDF Templates</h3>
+                  <PremiumFeatureButton
+                    canUseFeature={canCreateMoreTemplates}
+                    buttonContent={{
+                      title: "Add template",
+                      message: upgradeMessage,
+                      ctaText: "upgrading to team plan",
+                    }}
+                    skipCta={tier.id === TierId.tier_2}
+                    buttonProps={{
+                      to: "new",
+                      role: "link",
+                      icon: "plus",
+                      "aria-label": `new template`,
+                      "data-test-id": "createNewTemplateButton",
+                      variant: "primary",
+                    }}
+                  />
+                </div>
+                <div className="mt-5 w-full flex-1 overflow-x-auto rounded-[12px] border bg-white">
+                  <Table>
+                    <ListHeader
+                      children={
+                        <>
+                          <Th>Default</Th>
+                          <Th>Status</Th>
+                          <Th>Actions</Th>
+                        </>
+                      }
+                    />
+                    <tbody>
+                      {items.map((template) => (
+                        <ListItem item={template} key={template.id}>
+                          <TemplateRow item={template} />
+                        </ListItem>
+                      ))}
+                    </tbody>
+                  </Table>
+                </div>
+              </>
+            )}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+type TTemplate = Pick<
+  Template,
+  "id" | "name" | "type" | "isDefault" | "isActive" | "pdfSize" | "pdfUrl"
+>;
+
+const TemplateRow = ({ item }: { item: TTemplate }) => (
+  <>
+    <Td className="w-full">
+      <div className="flex flex-col items-start justify-between">
+        <span className="text-text-sm font-medium text-gray-900">
+          {item.name}
+        </span>
+        <span className="text-text-sm font-light text-gray-600">
+          {item.type}
+        </span>
+      </div>
+    </Td>
+    <Td>
+      {item.isDefault && (
```

Comment:
Lets add also a badge for disabled. Text should be "Inactive" and lets Make it gray. You can find some example of the gray color we use for badges in the code.
<img width="912" alt="Screenshot 2023-12-07 at 14 07 25" src="https://github.com/Shelf-nu/shelf.nu/assets/7840007/16c0bdc3-9e52-4055-846e-80040dbcc704">